app/main.py

```diff
@@ -2,0 +1,19 @@
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3 and command.split()[0] == "mv":
+        mv, file1_name, destination = command.split()
+        dirs = destination.split("/")[:-1]
+        file2_name = destination.split("/")[-1]
```

Comment:
The code does not handle the case where the destination path ends with a slash, which should be considered a directory. You need to check if `destination` ends with a '/' and handle it accordingly.